torch/_meta_registrations.py

```diff
@@ -2649,4 +2649,9 @@ def activate_meta():
                 _meta_lib_dont_use_me_use_register_meta.impl(op_overload, fn)
 
 
+@register_meta(aten.all_reduce)
+def all_reduce_meta(self, reduceOp, tag, rankset, stride):
+    return self
```

Comment:
this is actually wrong, i updated my PR to have a correct version.  it needs to be empty_like(self), or it traces out wrong code that isn't functional.

Reply:
👍